trifid/templates/service.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "trifid.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ template "trifid.name" . }}
+    helm.sh/chart: {{ template "trifid.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: {{ .Values.service.port }}
+      targetPort: http
+      protocol: TCP
+      name: http
+  selector:
+    app.kubernetes.io/name: {{ template "trifid.name" . }}
```

Comment:
Same issue regarding the labels as mentioned before

Reply:
Added the `helm create` defaults. I was running an outdated helm version... 🤦‍♂